src/metaswitch/clearwater/etcd_shared/plugin_utils.py

```diff
@@ -62,15 +62,21 @@ def run_command(command, namespace=None, log_error=True):
     stdout, stderr = p.communicate()
     if p.returncode != 0:
         # it failed, log the return code and output
-        _log.error("Command {} failed with return code {}"
-                   " and printed output {!r}".format(command,
-                                                     p.returncode,
-                                                     p.output))
-        return p.returncode
+        if log_error:
+            _log.error("Command {} failed with return code {}, "
+                       "stdout {!r}, and stderr {!r}".format(command,
+                                                             p.returncode,
+                                                             p.stdout,
+                                                             p.stderr))
```

Comment:
As @rkd pointed out, this is broken.
p.stdout and p.stderr should be changed to just stdout and stderr, as set by p.communicate() above.